### PR TITLE
GDPR changes

### PR DESCRIPTION
**Pull request by amakarudze**
I haven't added content to the pages yet but this is my suggestion for the GDPR chapter and pages as per this guideline https://medium.com/ft-product-technology/a-developers-guide-to-gdpr-that-won-t-make-you-sweat-4f1f7f1d9c8b.

Feedback very much appreciated.

### CODE REVIEW

File: gdpr/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+ 
+It regulates the use of data by organisations.
+It applies to any organisation that stores or processes
+ any data relating to individuals inside the European 
+ Union (EU). 
+ 
+ Since we have events in Europe, it is important 
+ for Django Girls Foundation and organizers across 
+ the globe to comply with GDPR as it has legal 
+ implications. We therefore recommend all organizers
```
Comment (asendecka) [inline gdpr/README.md]: I wonder if "recommend" is not too "soft" word. Should we write instead: All organizers should comply with GDPR

File: gdpr/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+Coaches and organizers need to be informed that 
+they should protect personal data of attendees and
+other coaches/organizers and not share it amongst
+themselves without the consent of the person 
+involved.
+
+# Subject Access Requests
+
+> Make it easy to retrieve, delete and anonymise 
+data when you need to.
```
Comment (asendecka) [inline gdpr/README.md]: Should we add the section here?

Comment (amakarudze) [reply gdpr/README.md]: I have added the "OK with Photo" or "Not OK with photo here" as many organizers seem to just publish photos without asking for consent, under the anonymise data so maybe we can still add that here. What do you think?

File: gdpr/README.md (hunk 1 above)
Comment (amakarudze) [reply gdpr/README.md]: Noted.

**Pull request merged**